projects/celerbridge/index.js

```diff
@@ -84,16 +428,28 @@ function chainTvl(chain) {
             if (token[chain] === undefined) {
                 return
             }
-            const balanceV1 = await sdk.api.erc20.balanceOf({
-                chain,
-                block,
-                target: token[chain],
-                owner: bridgeContractV1
-            })
+            let balanceV1;
+            try {
+                balanceV1 = await sdk.api.erc20.balanceOf({
+                    chain,
+                    block: block,
+                    target: token[chain],
+                    owner: bridgeContractV1
+                })
+            } catch (error) {}
```

Comment:
better to fail hard in case this fails

Reply:
Was getting `CALL_EXCEPTION`s pretty frequently but not deterministically. Any suggestion?